README.md

```diff
@@ -33,7 +33,7 @@ This documentation is aimed to help prospective node-operators set up and config
   - [Testnet Upgrade to casper-node v1.2.0](/docs/testnet/upgrade-1_2_0.md)
   - [Testnet Upgrade to casper-node v1.2.1](/docs/testnet/upgrade-1_2_1.md)
   - [Testnet Upgrade to casper-node v1.3.1](/docs/testnet/upgrade-1_3_1.md)
-  - [IMPORTANT - Testnet Upgrade to casper-node v1.3.2 before August 05](/docs/testnet/upgrade-1_3_2.md)
```

Comment:
1.3.2 instructions should not be removed, but rather reformatted to a required upgrade before the **IMPORTANT** upcoming upgrade notice.

Reply:
Indeed. Thanks for catching that!